llvm/lib/Analysis/ValueTracking.cpp

```diff
@@ -5805,6 +5805,36 @@ void computeKnownFPClass(const Value *V, const APInt &DemandedElts,
 
     break;
   }
+  case Instruction::BitCast: {
+    const Type *Ty = Op->getType();
+    const Value *Casted = Op->getOperand(0);
```

Comment:
```suggestion
    const Value *Src = Op->getOperand(0);
```
`Casted` sounds like the result of a cast inst.